src/Security/Authentication/Negotiate/src/NegotiateHandler.cs

```diff
@@ -328,6 +329,19 @@ protected override async Task<AuthenticateResult> HandleAuthenticateAsync()
             {
                 Principal = user
             };
+
+            if (Options.LdapSettings.EnableLdapClaimResolution)
+            {
+                await Events.RetrieveLdapClaims(authenticatedContext);
```

Comment:
Hmm, I'd need access to the LdapSettings in the event, but now they're internal on the options. Time to make a new ldap event context and pass in the ldap settings.

Reply:
Ah yes, I forgot about this when I made the settings internal.